src/main/java/hu/oe/nik/szfmv/automatedcar/powertrain/ITransmission.java

```diff
@@ -0,0 +1,29 @@
+package hu.oe.nik.szfmv.automatedcar.powertrain;
+
+/**
+ * ITransmission interface
```

Comment:
Avoid repeating information, that is obvious, like the type name and its format (interface).